core/src/mindustry/game/SpawnGroup.java

```diff
@@ -38,10 +40,8 @@
     /** Amount of enemies spawned initially, with no scaling */
     public int unitAmount = 1;
     /** Status effect applied to the spawned unit. Null to disable. */
-    @Nullable
```

Comment:
You're using an old commit. There's a bunch of reverted changes in this file.

Reply:
ohno nonononononoononooooooooooooooooooo

Reply:
lmao

Reply:
:/